app/src/main/java/com/android/unio/ui/association/AssociationProfile.kt

```diff
@@ -225,98 +253,165 @@ fun AssociationRecruitment(context: Context) {
   }
 }
 
+/**
+ * Component that display the users that are in the association that can be contacted. It display
+ * the title of the section and then display the different users in the association.
+ *
+ * @param userList (List<User>) : The list of users in the association that can be contacted
+ * @param context (Context) : The context of the screen
+ */
 @Composable
-fun UserCard(context: Context) {
+private fun UsersCard(userList: List<User>, context: Context) {
   Text(
-      getString(R.string.association_contact_members),
+      context.getString(R.string.association_contact_members),
       style = AppTypography.headlineMedium,
       modifier = Modifier.padding(horizontal = 20.dp).testTag("AssociationContactMembersTitle"))
   Spacer(modifier = Modifier.size(4.dp))
-  Box(
-      modifier =
-          Modifier.testTag("AssociationContactMembersCard")
-              .padding(horizontal = 23.dp)
-              .width(366.dp)
-              .height(40.dp)
-              .background(Color.LightGray, RoundedCornerShape(12.dp))
-              .padding(vertical = 2.dp, horizontal = 3.dp)
-              .clickable {
-                scope!!.launch {
-                  testSnackbar!!.showSnackbar(
-                      message = DEBUG_MESSAGE, duration = SnackbarDuration.Short)
-                }
-              },
-  ) {
-    Row(
-        horizontalArrangement = Arrangement.spacedBy(115.dp, Alignment.Start),
-        verticalAlignment = Alignment.CenterVertically,
+  userList.forEach { user ->
+    Box(
+        modifier =
+            Modifier.padding(horizontal = 23.dp)
+                .width(366.dp)
+                .height(40.dp)
+                .background(Color.LightGray, RoundedCornerShape(12.dp))
+                .padding(vertical = 2.dp, horizontal = 3.dp)
+                .clickable {
+                  scope!!.launch {
+                    testSnackbar!!.showSnackbar(
+                        message = DEBUG_MESSAGE, duration = SnackbarDuration.Short)
+                  }
+                },
     ) {
-      Icon(Icons.Filled.Person, contentDescription = "user's profile picture", Modifier.size(36.dp))
-      Text(text = "<Casey Rue>", style = AppTypography.headlineSmall)
+      Row(
+          horizontalArrangement = Arrangement.spacedBy(115.dp, Alignment.Start),
+          verticalAlignment = Alignment.CenterVertically,
+      ) {
+        Icon(
+            Icons.Filled.Person,
+            contentDescription = "user's profile picture",
+            Modifier.size(36.dp))
```

Comment:
User has a profilePicture value, why not directly implement it ? maybe it could be a small separate task, but then I fell like we should add it to the scrum board for this sprint